src/alr/gnat-command_line-extra.ads

```diff
@@ -0,0 +1,11 @@
+package GNAT.Command_Line.Extra is
```

Comment:
I am not sure if it is a good idea to make add a child package to a run-time unit.

Reply:
I could not think of other "lightweight" solution... I considered as option B to have a wrapper on GNAT.Command_Line.Command_Line_Configuration, but I am not sure if it's worth the effort.

Any other ideas?